rebasehelper/cli.py

```diff
@@ -164,6 +178,9 @@ def run():
         except RebaseHelperError as e:
             logger.error('\n%s', six.text_type(e))
             sys.exit(1)
+        except EnableOptionFormatException as e:
+            sys.stderr.write("{}".format(e))
+            sys.exit(1)
```

Comment:
I that case please think about moving the check and raising the exception after the parsing of arguments. Also the logging is little bit reworked in another PR which we will be merging shortly and it may help you with the problem.